docs/ADR/01_Opprettelse av faktureringskomponenten.md

```diff
@@ -0,0 +1,55 @@
+# Opprettelse av faktureringskomponenten
+
+## Status
+Akseptert
+
+## Kontekst
+Vi har fått i oppdrag å fase ut avgiftssystemet (legacy løsning), for å hjelpe til med avvikling av stormaskin. 
+Vi har i samråd med team Løst, som eier en fakturaløsning implementert med Oracle E-Business Suite (OEBS), kommet frem til 
+at vi skal koble oss på OEBS-løsningen for fakturering av trygdeavgift i de tilfellene der bruker skal betale avgiften til NAV.
+
+Gjennom dette beslutningsgrunnlaget ønsker vi å komme frem til hvordan samhandling mellom Melosys og OEBS skal foregå, 
+og hvem som har ansvar for de ulike funksjonene og som eier de data som er tilknyttet. 
+
+Grunnlaget gjelder i hovedsak:
+1) hvilken løsning som skal ha ansvar for at fakturereringen foregår regelmessig i hele vedtaksperioden, og
+2) hvem som har ansvar for å beregne endringer for neste faktura ved endringer i midt i en faktureringsperiode eller tilbake i tid.
+
+
+## Beslutning
+Ansvarsfordeling blir følgende:
+
+### Melosys-api
+Har ansvar for og eier data for vedtak og perioder det skal betales trygdeavgift for. 
+Melosys eier også endringer i vedtaket, herunder endringer i perioder, sats og beløpet som gjelder trygdeavgift.
+Melosys sender informasjon om vedtak og trygdeavgift som skal faktureres til faktureringskomponent som faktureringsoppdrag.
+Melosys har ansvar for å støtte saksbehandling av melding om manglende innbetaling fra OEBS.
+
+### Faktureringskomponent
+- Har ansvar for å holde i gang regelmessig bestilling av faktura for hele vedtaksperioden.
+- Sender bestilling av faktura til OEBS.
+- Håndterer endringer og motregninger i framtidige faktura opp mot tidligere faktureringsoppdrag.
+- Blir satt opp slik at den kan brukes til flere formål og ikke bare til fakturering av trygdeavgift. 
+
+### OEBS
+- Sending av faktura til den enkelte bruker, håndtering av innbetalinger og purring av manglende innbetalinger.
+OEBS eier all informasjon om det som er fakturert og det som er innbetalt. Dette blir ikke sendt tilbake og blir ikke duplisert slik som i dagens løsning.
+Dette medfører at OEBS må sende beskjed om manglende innbetaling etter purringer og frist. 
+Dette i henhold til felles gjeldende skatteregler, eller det som er ønskelig fra fagområde.
```

Comment:
Skal vi nevne hvordan løsningen snakker med Melosys? Altså at kommunikasjonen mellom OEBS og Melosys foregår over Kafka?

Reply:
Ja, det er ønskelig. Tenker dette er en egen ADR (som nevnt i https://jira.adeo.no/browse/MELOSYS-5506). 

Jeg kan opprette den men trenger hjelp for innhold siden jeg ikke var med.